cpp/src/lance/format/manifest.cc

```diff
@@ -85,4 +85,6 @@ void Manifest::AppendFragments(const std::vector<std::shared_ptr<DataFragment>>&
   fragments_.insert(std::end(fragments_), std::begin(fragments), std::end(fragments));
 }
 
+arrow::DatasetVersion Manifest::ToDatasetVersion() const { return arrow::DatasetVersion{version_}; }
```

Comment:
nit: not sure what's the right C++ convention but here's we're not turning a Manifest into a DatastVersion, instead we're getting it's version_ as a Datasetversion right? Isn't GetDatasetVersion more appropriate?

Reply:
sgtm. will change.